modules/33acrossIdSystem.md

```diff
@@ -41,7 +41,7 @@ The following settings are available for the `storage` property in the `userSync
 | --- | --- | --- | --- | --- |
 | name | Required | String| Name of the cookie or HTML5 local storage where the user ID will be stored | `"33acrossId"` |
 | type | Required | String | `"html5"` (preferred)  or `"cookie"` | `"html5"` |
-| expires | Strongly Recommended | Number | How long (in days) the user ID information will be stored. 33Across recommends `90`. | `90` |
+| expires | Strongly Recommended | Number | How long (in days) the user ID information will be stored. 33Across recommends `30`. | `30` |
```

Comment:
Change looks good, please also make the change on the following repo:
https://github.com/prebid/prebid.github.io/blob/master/dev-docs/modules/userid-submodules/33across.md